pkg/local_object_storage/shard/writecache.go

```diff
@@ -38,9 +36,5 @@ func (s *Shard) FlushWriteCache(p FlushWriteCachePrm) error {
 		return ErrDegradedMode
 	}
 
-	if err := s.writeCache.SetMode(mode.ReadOnly); err != nil {
```

Comment:
the desc should be fixed then. btw, how does that relate to the commit msg? is it a necessary change?

Reply:
Fixed description, yes it is confusing, see linked bug.